tracker/tracker/src/main/app/index.ts

```diff
@@ -81,6 +83,8 @@ export default class App {
   private activityState: ActivityState = ActivityState.NotActive;
   private version = 'TRACKER_VERSION'; // TODO: version compatability check inside each plugin.
   private readonly worker?: Worker;
+  private localStorage: Storage;
+  private sessionStorage: Storage;
```

Comment:
Could you please make them  `readonly` instead of `private` in order to allow using this api for plugins  as well?